autogluon/utils/tabular/ml/models/fasttext/hyperparameters/parameters.py

```diff
@@ -0,0 +1,17 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: m.ml.models.fasttext.hyperparameters.parameters.ipynb (unless otherwise specified).
+
+__all__ = ["get_param_baseline"]
+
+# Cell
+# https://fasttext.cc/docs/en/python-module.html#train_supervised-parameters
+def get_param_baseline():
+    params = {
+        "lr": 0.1,
+        "epoch": 10,
```

Comment:
Is 10 too small for default? Will this model tend to overfit?

Reply:
I do not have extensive testing of the best default value for number of epochs. The 10 is mostly for not getting too slow on big datasets.

In the income class dataset, here is are some test runs. Maybe I change the default to 50. 

- epoches: 10, train accurary 0.830906, test accurary 0.838878
- epoches: 50, train accurary 0.832339, test accurary 0.841028
- epoches: 100, train accurary 0.832391, test accurary 0.840925
- epoches: 200, train accurary 0.832851, test accurary 0.841335

Reply:
Thanks for the experiment!  I agree 50 seems like a more reasonable default than 10 although it would be nice to explore values like 20/30 as well with more datasets.